skimage/measure/_find_contours_cy.pyx

```diff
@@ -3,7 +3,12 @@
 #cython: nonecheck=False
 #cython: wraparound=False
 import numpy as np
+cimport numpy as np
```

Comment:
My personal preference is to differentiate the cimport by using `cimport numpy as cnp`. But using `np` is frequently used as well so you can leave it if you want.